flake.nix

```
@@ -165,10 +165,6 @@
       system: let
         overlays = [
           poetry2nix.overlay
-          rust-overlay.overlays.default
-          (self: super: {
-            npmlock2nix = pkgs.callPackage npmlock2nix {};
-          })
         ];
 
         pkgs = import nixpkgs {
```

Lets change this to use legacyPackages. We will save some evaluation time this way.

```suggestion
        pkgs = nixpkgs.legacyPackages.${system};
```